app/webpacker/components/CompetitionForm/FormSections/CompetitorLimit.js

```diff
@@ -18,6 +18,9 @@ export default function CompetitorLimit() {
         <InputNumber id="count" min={0} />
         <InputTextArea id="reason" />
       </ConditionalSection>
+      <ConditionalSection showIf={hasLimit}>
+        <InputNumber id="autoCloseThreshold" min={1} nullable />
+      </ConditionalSection>
```

Comment:
I feel like these two can be merged into the same `ConditionalSection`

Reply:
(both have `hasLimit` exactly set as their `showIf`)

Reply:
Updated